integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -34,20 +46,95 @@ export type AppConfigurationRelaxed = {
   baseFilePath?: string;
 };
 
+function getCredentials(): Credentials {
+  if (typeof publicKey === "string" && typeof privateKey === "string") {
+    return {
+      kind: "api-key",
+      publicKey,
+      privateKey,
+    };
+  } else {
+    return {
+      kind: "username-password",
+      username,
+      password,
+    };
+  }
+}
+
+const credentials = getCredentials();
```

Comment:
Looks like the function is only used once, so it may not need to be a function.
```suggestion
const credentials: Credentials =
  typeof publicKey === "string" && typeof privateKey === "string"
    ? { kind: "api-key", publicKey, privateKey }
    : { kind: "username-password", username, password };
```